pkg/scheduler/framework/interface.go

```diff
@@ -109,6 +109,10 @@ type Status struct {
 	code    Code
 	reasons []string
 	err     error
+	// failedPlugin is an optional field that records the plugin name a Pod failed by.
```

Comment:
update the comments on the `Status` struct to include this new optional field?

Reply:
done.